README.rst

```diff
@@ -1,62 +1,16 @@
-A Simple Python Project Skeleton
```

Comment:
@ziadhany please add some steps here that you want us to use to install this and run this locally

Reply:
All installation steps are in `docs/source/contribute/installation.rst`
So I added a quick Installation to `README.rst`